src/components/users/LoginForm/LoginForm.js

```diff
@@ -1,27 +1,17 @@
 import "./login-form.scss";
-import "./../../../style/common/common.scss";
+import "style/common/_common.scss";
 
 const LoginForm = () => (
   <form className="login-form">
-    <label className="login-form-input-container">
+    <label>
       Email
-      <input
-        className="login-form-input regular-text"
-        type="email"
-        placeholder="Type your email"
-      />
+      <input type="email" placeholder="Type your email" />
     </label>
-    <label className="login-form-input-container">
+    <label>
       Password
-      <input
-        className="login-form-input regular-text"
-        type="password"
-        placeholder="Type your password"
-      />
+      <input type="password" placeholder="Type your password" />
     </label>
-    <button type="button" className="common-black-button regular-text">
-      Log in
-    </button>
+    <button type="button">Log in</button>
```

Comment:
Why are you removing those css classes for all of your HTML tags?

Reply:
Because I have defined nested elements in the scss file

Reply:
It's not recommended to add styles to a specific element HTML tag inside the DOM hierarchy because if you add a new element to the hierarchy or change something in your HTML structure, your styles could break. 

It's better to use different classes and apply styles to those classes instead of add styles to HTML tags inside another tags